config/config.php

```diff
@@ -121,6 +121,23 @@ public function set_default_config()
 				'32'	=> 0,
 			)),
 
+			// Extension/style validation queue forums
+			'titania_extensions_queue'	=> array('default' => array(
+				ext::TITANIA_QUEUE_DISCUSSION	=> 0,
+				ext::TITANIA_QUEUE				=> 0,
+				'trash'							=> 0,
```

Comment:
Maybe we need a constant for this too? Seems strange to have constants for everything but this.